networksession.go

```diff
@@ -395,10 +356,25 @@ func (s *sessionTCP) exchangeTimeout(req Message, writeDeadline, readDeadline ti
 		}
```

Comment:
Please could you use `sync.Map` instead?

Reply:
For TCP yes but for UDP it is no possible because it is map in map and to delete entry i need lock whole nested objects. Then I will prefer same implementation for bot tcp and udp.